com.unity.ml-agents/Runtime/Sensors/ISensor.cs

```diff
@@ -59,16 +59,6 @@ public enum ObservationType
         /// Collected observations contain goal information.
         /// </summary>
         Goal = 1,
-
```

Comment:
We were going to rename "Goal" to "GoalSignal" too, right?

Reply:
I forgot this conversation. :( I can change it!